kafka/tools/assigner/sizers/prometheus.py

```diff
@@ -0,0 +1,125 @@
+import re
+try:
+    from urllib.request import urlopen
+except ImportError:
+    from urllib import urlopen
+
+from kafka.tools import log
+from kafka.tools.exceptions import UnknownBrokerException, ConfigurationException
+from kafka.tools.assigner.sizers import SizerModule
+
+
+class SizerPrometheus(SizerModule):
+    name = 'prometheus'
+    helpstr = 'Get partition sizes by connection to each broker via Proemtheus metrics'
```

Comment:
Looks like "Proemtheus" might be spelled wrong here. Also, since some folks might not know what prometheus is, it might be good to make a reference to the URL: "https://prometheus.io/".